wifiphisher/macmatcher.py

```diff
@@ -0,0 +1,45 @@
+import os
+from constants import *
```

Comment:
Can you change this to
```python
import constants
```
This way is cleaner and shows exactly what variables are from constants.